packages/circuits/README.md

```diff
@@ -106,7 +106,7 @@ import { Circuit } from '@unirep/circuits'
 // See ./test/verifyEpochKey.test.ts for generating circuit inputs
 const circuitInputs = {
     state_tree_elements: ...,
-    state_tree_indexes: ...,
+    state_tree_indeces: ...,
```

Comment:
I think this should be either `indexes` or `indices`

Reply:
I think it should match the epoch tree and history tree
https://github.com/Unirep/Unirep/blob/508ebd6c8f3366f923a1657c0800dad2a5a7eb45/packages/circuits/circuits/userStateTransition.circom#L53
https://github.com/Unirep/Unirep/blob/508ebd6c8f3366f923a1657c0800dad2a5a7eb45/packages/circuits/circuits/userStateTransition.circom#L36